src/InMemoryRepository.php

```diff
@@ -81,7 +81,7 @@ public function find($query, $language = 'glob')
         $resources = array();
 
         if (Glob::isDynamic($query)) {
-            $resources = $this->getGlobIterator($query);
+            $resources = iterator_to_array($this->getGlobIterator($query));
```

Comment:
ArrayResourceCollection currently requires both `Traversable` and `ArrayAccess` (afaik), but there isn't an interface extending both them (and the creation just to fill a PHPDoc seems useless to me), so I would suggest to revert this change and just ignore the Scrutinizer issue until next changes to the ArrayResourceCollection.

What to you think? Did I miss something?